src/librustc_typeck/collect.rs

```diff
@@ -2883,6 +2901,25 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, id: DefId) -> CodegenFnAttrs {
         }
     }
 
+    let no_sanitize_flags = CodegenFnAttrFlags::NO_SANITIZE_ADDRESS
+        | CodegenFnAttrFlags::NO_SANITIZE_MEMORY
+        | CodegenFnAttrFlags::NO_SANITIZE_THREAD;
```

Comment:
I am not crazy about this enumeration being here -- it seems very easy to forget to add something to it. Can we somehow make this more DRY?

Reply:
Replaced with `NO_SANITIZE_ANY`.